### DISCUSSION TITLE
Lost my access to the authenticator.

### DISCUSSION THREAD

**Discussion by Zeusangis** [community/community#137292] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

So, I lost access to my authenticator and all other way to 2FA as my old phone was gone and bought new one. Without 2FA I am unable to access my GitHub. I have several websites there and also has a really important second email connected to it. I however have access to the both primary and secondary email. Please let me know if I can do something about it.

**Comment by davevad93** [comment, marked as answer]
Hi @Zeusangis ,

Considering it's account related,, I think that by contacting support you have a better chance of solving your problem, open a ticket [here](https://support.github.com/contact/account), select `Two factor authentication recovery` or `Device verification on sign in, unable to access email` and explain your issue.

**Reply by queenofcorgis** [reply]
Please open a ticket like @davevad93 suggested. Please note: the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
